build/test-replacements.js

```diff
@@ -106,6 +112,18 @@ module.exports['common.js'] = [
         + '}\n'
         + '/*</replacement>*/\n'
     ]
+  , [
```

Comment:
this isn't needed in the browser and cause some issues with browsers that don't have uint8clampedarray